stores/ip-based-access/store.fga.yaml

```diff
@@ -0,0 +1,79 @@
+model: |
+  model
+    schema 1.1
+
+  type user
+
+  type organization
+    relations
+      define member : [user]
+      define ip_based_access_policy : [organization#member with in_company_network]
+
+  type document
+    relations
+      define organization : [organization]      
+      define viewer: [user]
+      
+  # document
```

Comment:
Was this intended? Seems we should remove this and collapse the spacing between `define viewer: [user]` and `define can_view: ...`